### PR TITLE
(GH-3) Add documentation

### LINKED ISSUE #3: Add documentation
We should add a website, created using Wyam, documenting the addin

### PR DESCRIPTION
**Pull request by pascalberger**
Add simple documentation showcasing the existing out of the box reports.

Fixes #3

### CODE REVIEW

File: docs/input/docs/index.cshtml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+Title: Documentation
+---
+<p>This user guide, like Cake.CodeAnalysisReporting itself, is under very active development. Some parts of it aren't 
+documented as completely as they need to be, but we glady accept your contributions.</p>
```
Comment (stffabi) [inline docs/input/docs/index.cshtml]: Typo: gladly

File: docs/input/docs/reports/msbuild/xmlfilelogger/msbuild-xmlfilelogger-by-assembly.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+---
+Order: 1
+Title: Grouped By Assembly
+---
+
+This report can be generated from a MsBuild log file genereated by the [XmlFileLogger] from the MSBuild Extension Pack
```
Comment (stffabi) [inline docs/input/docs/reports/msbuild/xmlfilelogger/msbuild-xmlfilelogger-by-assembly.md]: Typo: generated

File: docs/input/docs/reports/msbuild/xmlfilelogger/msbuild-xmlfilelogger-by-rule.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+---
+Order: 2
+Title: Grouped By Rule Number
+---
+
+This report can be generated from a MsBuild log file genereated by the [XmlFileLogger] from the MSBuild Extension Pack
```
Comment (stffabi) [inline docs/input/docs/reports/msbuild/xmlfilelogger/msbuild-xmlfilelogger-by-rule.md]: Typo: generated

**Review by stffabi — APPROVED** [review]
Some typos, the remaining LGTM.

File: docs/input/docs/index.cshtml (hunk 1 above)
Comment (pascalberger) [reply docs/input/docs/index.cshtml]: @gep13 I think this typo is also in all other repos :)

File: docs/input/docs/reports/msbuild/xmlfilelogger/msbuild-xmlfilelogger-by-assembly.md (hunk 2 above)
Comment (pascalberger) [reply docs/input/docs/reports/msbuild/xmlfilelogger/msbuild-xmlfilelogger-by-assembly.md]: Fixed

File: docs/input/docs/reports/msbuild/xmlfilelogger/msbuild-xmlfilelogger-by-rule.md (hunk 3 above)
Comment (pascalberger) [reply docs/input/docs/reports/msbuild/xmlfilelogger/msbuild-xmlfilelogger-by-rule.md]: Fixed

**Pull request merged**